packages/Table/stories/Table.colspans.stories.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { getStoryName } from "storybook/storyTree";
+import Table, { TH } from "../src";
+
+const storyName = getStoryName("Table");
+
+storiesOf(storyName, module);
+
+const data = [
+  { name: "Charles ", lastName: "Babbage", iceCream: ["vanilla", "chocolate"] },
+  { name: "Claude ", lastName: "Shannon", iceCream: ["lime"] },
+  { name: "Alan ", lastName: "Turing", iceCream: ["strawberry"] },
+  { name: "John  von", lastName: " Neumann", iceCream: ["blueberry"] },
+  { name: "William", lastName: "Shockley", iceCream: ["Chocolate chip"] },
+  { name: "Douglas", lastName: "Engelbart", iceCream: ["Neopolitan", "Coffee"] },
+  { name: "Robert", lastName: "Noyce", iceCream: ["Green tea", "Grape"] },
+  { name: "Steve", lastName: "Wozniak", iceCream: ["Mango"] },
+  { name: "Grace Murray", lastName: "Hopper", iceCream: ["Raspberry Ripple"] },
+  { name: "Vint", lastName: "Cerf", iceCream: ["Teaberry "] },
+];
+
+export const WithColSpan = () => {
+  return (
+    <Table a11yText="" data={data}>
+      <Table.Headers>
+        <tr>
+          <TH colSpan="3">List of ice creams</TH>
+        </tr>
+      </Table.Headers>
```

Comment:
Can we include this detail that the `children` of `<Table.Headers>` should include a `<tr>` in the Docs MDX page?  Probably just a sentence or two at the bottom of that page under "Custom Headers" would be enough.

Reply:
added documentation, can you take a look :)